corehq/apps/settings/views.py

```diff
@@ -643,22 +671,28 @@ def page_context(self):
     @property
     def paginated_list(self):
         for api_key in self.base_query.order_by('-created').all()[self.skip:self.skip + self.limit]:
-            redacted_key = f"{api_key.key[0:4]}…{api_key.key[-4:]}"
             yield {
-                "itemData": {
-                    "id": api_key.id,
-                    "name": api_key.name,
-                    "key": redacted_key,
-                    "domain": api_key.domain or _('All Projects'),
-                    "ip_allowlist": (
-                        ", ".join(api_key.ip_allowlist)
-                        if api_key.ip_allowlist else _("All IP Addresses")
-                    ),
-                    "created": api_key.created.strftime('%Y-%m-%d %H:%M:%S'),
-                },
+                "itemData": self._to_json(api_key),
                 "template": "base-user-api-key-template",
             }
 
+    @staticmethod
+    def _to_json(api_key):
+        redacted_key = f"{api_key.key[0:4]}…{api_key.key[-4:]}"
+        return {
+            "id": api_key.id,
+            "name": api_key.name,
+            "key": redacted_key,
+            "domain": api_key.domain or _('All Projects'),
+            "ip_allowlist": (
+                ", ".join(api_key.ip_allowlist)
+                if api_key.ip_allowlist else _("All IP Addresses")
+            ),
+            "created": api_key.created.strftime('%Y-%m-%d %H:%M:%S'),
```

Comment:
It'd be good to display this in the user's/project's time zone (depending on where in the UI this is, I think this is a user-specific page not a project-specific page), not UTC.

Here's an example of manipulating time zones for a UI: https://github.com/dimagi/commcare-hq/pull/32258/files#diff-cfe7f189c4e34aef131f2357f43abbed31f40d0de103db5cbe71880f4436115dR697-R700

Reply:
Thoughts on this approach?
![image](https://user-images.githubusercontent.com/2367539/200634510-0afdd4ec-c9f0-4a46-80d9-cf78a7043a25.png)

That uses `corehq.const.USER_DATETIME_FORMAT`, which includes a timezone string.